doc/whats_new.rst

```diff
@@ -18,7 +18,9 @@ parameters, may produce different models from the previous version. This often
 occurs due to changes in the modelling logic (bug fixes or enhancements), or in
 random sampling procedures.
 
-* *to be listed*
+   - Made a change to :class:`sklearn.ensemble.IsolationForest` by 
```

Comment:
I think you'll just have to list the class `:class:`sklearn.ensemble.IsolationForest` and the user is expected to ctrl-f it out... Confirm with @jnothman though...